tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+    "compilerOptions": {
```

Comment:
You should probably also add:
```json
{
  "strict": true,
}
```
It'll enable a bunch of crap for you to ensure stricter typing, which is usually a good idea.